tests/drivers/spi/spi_loopback/boards/96b_carbon.conf

```diff
@@ -1 +1,2 @@
 CONFIG_SPI_LOOPBACK_DRV_NAME="SPI_2"
+CONFIG_SPI_STM32_INTERRUPT=y
```

Comment:
I wonder why this is required, as any STM32 target, `overlay-stm32-spi.conf` should be applied.

Reply:
no, the `config SPI_STM32_INTERRUPT`  is set for each board upon  `depends on SPI`

Reply:
Right, my bad.